### ISSUE TITLE
A11y_SQL Azure Data Studio_New deployment -SQL Server container image_Keyboard:Tooltips of Collapse Cells, Clear Result ,Trusted etc buttons is not accessible with keyboard.

### ISSUE THREAD

**Issue by sanket856** [microsoft/azuredatastudio#15844]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
Accessibility Insights
Solve accessibility issues before they reach your customers.

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.1.1;#Keyboard;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.30.0-insider (user setup)
Windows Version: Windows10
Screenreader: JAWS, NVDA

### Repro Steps:

1.  Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hit enter on the ellipses '...' and select "New Deployment"
4.  In the Deployment options, select "SQL Server container image" card.
5.  Validate the required tools are installed and the "Required Tools" table shows the status as "Installed". Without the required tools the Select button will be disabled.
6. Then hit enter on "Select".
7. Fill in password details and click "Open Notebook".
8. A new Notebook in ADS containing the script to deploy a SQL Server container image should be displayed. User should be able to click "Run all cells" in the Notebook to deploy the container image successfully.
9. Then using tab key navigate Collapse Cells, Clear Result ,Trusted buttons.
10. Observe that tooltips of Collapse Cells, Clear Result ,Trusted etc. buttons is not accessible with keyboard.

### Actual:
Tooltips of Collapse Cells, Clear Result ,Trusted etc. buttons is not accessible with keyboard.

### Expected:
Tooltips of Collapse Cells, Clear Result ,Trusted etc. buttons should be accessible with keyboard.

### User Impact:
Keyboard only user will face difficulty while accessing the controls as they wont be able to access the tooltip with keyboard.

**Comment by chlafreniere** [comment]
This is a general problem from vscode: https://github.com/microsoft/vscode/issues/52016
